src/modules/gnd_pos_control/gnd_pos_control_main.cpp

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
  *
- *   Copyright (c) 2013-2016 PX4 Development Team. All rights reserved.
+ *   Copyright (c) 2013-2017 PX4 Development Team. All rights reserved.
```

Comment:
Since this is new code, you can put 2017 only, even if it is based on an existent module.